pages/cohort-enrollment/[enrollmentCode].vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="container">
+    <div v-if="pending">Enrolling in the cohort...</div>
+    <div v-else>
+      <div v-if="cohort">You have been enrolled in the cohort!</div>
+      <div v-else>
+        <div>
+          The cohort does not exist or you do not have access to enroll.
+        </div>
+      </div>
```

Comment:
Do you need this nested `divs` or could we do:

```suggestion
      <div v-else>
        The cohort does not exist or you do not have access to enroll.
      </div>
```

Reply:
We can do that!